src/components/practices/funny_practices/DragDropSentenceBuilder.jsx

```
@@ -1,258 +1,315 @@
+          <div className="mb-6 p-3 bg-blue-50 border border-blue-200 rounded-lg">
+            <p className="font-medium">Sentence Type: Present perfect continuous</p>
+            <p className="text-sm text-gray-600">Drag and drop the words to form a correct sentence.</p>
+          </div>
+          <DropZone sentence={sentence} onDrop={handleDrop} onRemove={handleRemove} />
+          <div className="mt-6 flex flex-wrap p-4 bg-gray-50 rounded-lg">
+            {availableWords.map(({ word, id }) => (
+              <DraggableWord key={id} word={word} id={id} />
+            ))}
+          </div>
+          {feedback && feedback.correct && (
+            <div className="p-4 mt-4 border-l-4 border-green-500 bg-green-50 rounded-md text-green-700">
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)